src/controllers/TalksController.php

```diff
@@ -124,9 +124,14 @@ public function postAction($request, $db)
                         $comment    = $comment_mapper->getCommentById($new_id);
                         $speakers   = $talk_mapper->getSpeakerEmailsByTalkId($talk_id);
                         $recipients = array();
+                        $commentAuthorEmail = $comment[0]['email'];
```

Comment:
I'd say it's more readable without the additional assignment 😄 

Directly in the foreach I find a variable that I need to check what it contains before I can decide what it does. With the direct comparison I'd see it's both an email. 

Additionally the new variable is only used in this one single line. For me that usually is a sign that the assignment can be omitted.